chemprop/cli/utils/parsing.py

```diff
@@ -403,12 +404,17 @@ def make_dataset(
     multi_hot_atom_featurizer_mode: str = "V2",
 ) -> MoleculeDataset | ReactionDataset:
     atom_featurizer = get_multi_hot_atom_featurizer(multi_hot_atom_featurizer_mode)
+    if multi_hot_atom_featurizer_mode == "RIGR":
+        bond_featurizer = RIGRBondFeaturizer()
+    else:
+        bond_featurizer = MultiHotBondFeaturizer()
```

Comment:
```suggestion
    match  multi_hot_atom_featurizer_mode 
        case "RIGR":
            bond_featurizer = RIGRBondFeaturizer()
        case "V2":
            bond_featurizer = MultiHotBondFeaturizer()
        case _:
           raise TypeError(f"Unsupported atom featurizer mode '{multi_hot_atom_featurizer_mode=}'!")
```

For future featurizers

Reply:
If the argument V1 or ORGANIC is passed, the bond_featurizer should still be `MultiHotBondFeaturizer()`.This updation doesn't ensure that, am I correct? Is it okay to have 4 cases for V1, V2, ORGANIC, and RIGR. The bond featurizer is the same for V1, V2, and ORGANIC, hence, tried to do it with `if .. elsel ..`

Reply:
Ah OK please update the type hint to reflect all these options and then just have one case like this:
```python
case "V1" | "V2" | ...:
```